Bio/PDB/parse_pdb_header.py

```diff
@@ -247,6 +296,9 @@ def _parse_pdb_header_list(header):
                 except:
                     # print('nonstandard resolution %r' % r)
                     dict['resolution'] = None
+            elif hh.startswith("REMARK 465"):
+                # Update the dictionary with content of the remark 465 line (Missing residues)
+                _parse_remark_465(tail, dict)
```

Comment:
`dict.update` would not work, because there are multiple "REMARK 465" lines. It would have to be something like:

    if tail:
        dict['has_missing_residues'] = True
        missing_res_info = _parse_remark_465(tail)
        if missing_res_info: #None, if we parsed a header-line of REMARK 465
             dict['missing_residues'].append(missing_res_info)